### ISSUE TITLE
Error when tokenizing Stripe element card

### ISSUE THREAD

**Issue by y3owk1n** [swellstores/swell-js#79]
I've posted this at the swell discussion, but I think this should come here instead.

When running the following code, I've ran into some error `"Missing value for stripe.confirmCardSetup intent secret: value should be a client_secret string."`

```
            await swell.payment.tokenize({
                card: {
                    onError: (err) => {
                        // inform the customer there was an error
                        console.log({ err });
                    },
                    onSuccess: async () => {
                        console.log("success");
                        //finally submit the form
                    },
                },
            });
```

Upon more debugging and testing, i found that the particular `confirmCardSetup` function does not take into consideration of camelCase response that causes this.

![image](https://user-images.githubusercontent.com/62775956/210380556-bec9e0ff-31a8-4fa1-9344-9c6f44384c2e.png)

---

Also, I did some patch locally and it works perfectly on camelCase. However, it doesn't seems to be attaching the tokenized card to the cart session as per [the doc](https://developers.swell.is/frontend-api/payments#tokenize-elements).

Upon checking the responses, I found a couple of things that are causing it, but i am not sure if it was intended or not.

Part 1: If capture amount is less than 1, then update the tokenized to the billing detail. (Obviously my capture amount is 52, and it wont be updated)

> Quoting the doc: In order to tokenize successfully, there must be a product in the cart with a grand total greater than 1.

![Screenshot 2023-01-03 at 11 27 35 PM](https://user-images.githubusercontent.com/62775956/210389523-0649f7dc-d050-4c1f-ac1d-534244b3bab9.png)

Part 2: If the `intent_status` is `requires_confirmation`, then it should confirm the card with Stripe and update the billing detail. (Obviously the response that I got back is `requires_capture`, and it never get updates)
![Screenshot 2023-01-03 at 11 32 31 PM](https://user-images.githubusercontent.com/62775956/210389938-0bd5124c-070c-41ff-b3f6-d8e93e48e9c5.png)

Up to this point the tokenization was a success but the cart session did not get any payment information attached. Wondering if it will only be attached automatically after submitting the order?

**Comment by gusfune** [comment]
This is the same issue that I have identified on #78, hopefully Swell might be fixing it soon.

**Comment by y3owk1n** [comment]
@gusfune its been 3 weeks since your report, hope it wont take too long to be fixed.

**Comment by rodrigoddalmeida** [comment]
@y3owk1n, @gusfune we're looking to push a fix real soon.

**Comment by swellmike** [comment]
The fix is now available in swell-js v3.18.3:
https://www.npmjs.com/package/swell-js
